assets/templates/carousel-component.html

```diff
@@ -74,18 +75,157 @@
         <div class="dpsg-content-wrap">
           <article class="dpsg-content">
             <h3>Carousel Component</h3>
-            <p>Nuevo componente de Carousel</p>
+            <p>
+              Al copiar el html del carousel cambiar <b>id</b> de cada
+              <b>slide</b>, siempre dejando la terminación <b>_dpslide</b>, y
+              poner un mismo <b>name</b> para todos los <b>slides</b> del
+              carousel.
+              <br />
+              Ejemplo: para el primer slide de un carousel
+              <b>id="primero_dpslide" name="carousel1"</b>. Para el segundo
+              slide <b>id="segundo_dpslide" name="carousel1"</b>, etc.
+              <br /><br />
+              Poner misma cantidad de inputs que de slides. A cada input ponerle
+              un <b>id</b> y <b>value</b> distinto, y el mismo <b>name</b> que a
+              los slides. <br />
+              Ejemplo: para el primer input de un carousel
+              <b>id="primero" value="primero" name="carousel1"</b> . Para el
+              segundo input <b>id="segundo" value="segundo" name="carousel1"</b>
+            </p>
             <hr class="dpsg-break" />
             <div class="dp-container">
               <div class="dp-rowflex">
-                <div class="col-sm-6 m-b-12">
-                  <div class="dp-box-shadow dpsg-content-sample">
-                    <p class="dp-boxshadow">Carousel 1</p>
+                <div class="col-sm-12 col-md-6">
+                  <div class="dp-carousel">
+                    <div class="dp-carousel-wrapper dp-carousel-orange">
+                      <div class="dp-carousel-content">
+                        <div
+                          class="dp-carousel-slide"
+                          id="primero_dpslide"
+                          name="carousel1"
+                        >
```

Comment:
You could use a class `active` in the slide that is supposed to show. So then you would have all the slides hidden by styles and change the class to active in the one that must be shown. 
```suggestion
                      <div class="dp-carousel-content">
                        <div
                          class="dp-carousel-slide active"
                          id="primero_dpslide"
                          name="carousel1"
                        >
```
This is more readable in the html, it has more semantic meaning.

Reply:
Of course you will need to add the classes for active and not active:
```
    .dp-carousel-content .dp-carousel-slide{
      display:none;
    }
    .dp-carousel-content .dp-carousel-slide.active{
      display:block;
    }
```

Reply:
Ok, I do it like this